docs/web/docs/3-reference/3-schemas/config.json

```diff
@@ -41,27 +41,27 @@
       "default": [
         {
           "type": "DIRECTORY_PATH",
-          "location": "/home/ccastrapel/localrepos/iambic/iambic/plugins/v0_1_0/aws",
+          "location": "/home/aboo-umair/Desktop/workspace/OpenSource/iambic/iambic/plugins/v0_1_0/aws",
```

Comment:
notice this is not good because I encode the full file path which is likely gonna keep changing depending who is generated the docs. I created another PR to fix the issue of full file path. please regenerate the docs after that PR is merged into main. 

https://github.com/noqdev/iambic/pull/620